Stripe.xcworkspace/xcshareddata/xcschemes/AllStripeFrameworks-iOS26.xcscheme

```diff
@@ -0,0 +1,297 @@
+               BlueprintIdentifier = "44C90013B7C82C80A2F69956"
+               BuildableName = "StripeFinancialConnections.framework"
+               BlueprintName = "StripeFinancialConnections"
+               ReferencedContainer = "container:StripeFinancialConnections/StripeFinancialConnections.xcodeproj">
+            </BuildableReference>
+         </BuildActionEntry>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "YES"
+            buildForProfiling = "YES"
+            buildForArchiving = "YES"
+            buildForAnalyzing = "YES">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "57AEC53510AE0DC0539730F3"
+               BuildableName = "Stripe3DS2.framework"
+               BlueprintName = "Stripe3DS2"
+               ReferencedContainer = "container:Stripe3DS2/Stripe3DS2.xcodeproj">
+            </BuildableReference>
+         </BuildActionEntry>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "YES"
+            buildForProfiling = "YES"
+            buildForArchiving = "YES"
+            buildForAnalyzing = "YES">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "41D17A3F2C5A73A6007C6EE6"
+               BuildableName = "StripeConnect.framework"
+               BlueprintName = "StripeConnect"
+               ReferencedContainer = "container:StripeConnect/StripeConnect.xcodeproj">
+            </BuildableReference>
+         </BuildActionEntry>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "NO"
+            buildForProfiling = "NO"
+            buildForArchiving = "NO"
+            buildForAnalyzing = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "41D17A492C5A73A7007C6EE6"
+               BuildableName = "StripeConnectTests.xctest"
+               BlueprintName = "StripeConnectTests"
+               ReferencedContainer = "container:StripeConnect/StripeConnect.xcodeproj">
+            </BuildableReference>
+         </BuildActionEntry>
+      </BuildActionEntries>
+   </BuildAction>
+   <TestAction
+      buildConfiguration = "Debug"
+      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
+      selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      shouldUseLaunchSchemeArgsEnv = "NO">
+      <PreActions>
+         <ExecutionAction
+            ActionType = "Xcode.IDEStandardExecutionActionsCore.ExecutionActionType.ShellScriptAction">
+            <ActionContent
+               title = "Run Script"
+               scriptText = "cd &quot;$SRCROOT/..&quot;&#10;echo &quot;Started at: $(date)&quot; | tee /tmp/generate_ios26_testplan.log&#10;ruby ci_scripts/generate_ios26_testplan.rb | tee -a /tmp/generate_ios26_testplan.log&#10;"
```

Comment:
I'm not seeing where we actually read in this log file -- if we don't ever read it back in, maybe it would be better to use mktemp to create a temp file?
```
scriptText = "cd &quot;$SRCROOT/..&quot;&#10;LOG_FILE=$(mktemp)&#10;echo &quot;Log file: $LOG_FILE&quot;&#10;echo &quot;Started at: $(date)&quot; | tee $LOG_FILE&#10;ruby ci_scripts/generate_ios26_testplan.rb | tee -a $LOG_FILE&#10;echo &quot;Log saved to: $LOG_FILE&quot;&#10;"
```